addons/base_import/models/base_import.py

```diff
@@ -1390,6 +1393,8 @@ def execute_import(self, fields, columns, options, dryrun=False):
 
         _logger.info('importing %d rows...', len(input_file_data))
 
+        input_file_data, additional_filenames = self._extract_additional_filenames(import_fields, input_file_data)
```

Comment:
I don't really like the "additional_filenames" terminology. What about "binary_filenames" or something in that spirit? This terminology spreads also in js (e.g. AdditionalFilenamesManager)